controllers/front/AddressController.php

```diff
@@ -59,17 +59,25 @@ public function init()
     public function postProcess()
     {
         $this->context->smarty->assign('editing', false);
-        $id_address = (int) Tools::getValue('id_address');
-        // Initialize address if an id exists
+
+        $params = Tools::getAllValues();
+        $id_address = isset($params['id_address']) ? (int) $params['id_address'] : 0;
```

Comment:
```suggestion
        $id_address = (int) Tools::getValue('id_address', 0);
```

you could do this

Reply:
I think this good, BUT we need pass array $params... 
Ok... I will test and improve this.

Reply:
no we need array of form post later... in fillform when form submit..